features/pull_request.feature

```diff
@@ -757,19 +757,44 @@ BODY
     Given I am on the "feature" branch with upstream "origin/feature"
     Given the GitHub API server:
       """
+      get('/repos/mislav/coral/milestones') {
+        status 200
+        json [
+          { :number => 1234, :title => "prerelease" },
+          { :number => 1337, :title => "v1" },
+          { :number => 41319, :title => "Hello World!" }
+        ]
+      }
       post('/repos/mislav/coral/pulls') {
         assert :head  => "mislav:feature"
         status 201
         json :html_url => "the://url", :number => 1234
       }
       patch('/repos/mislav/coral/issues/1234') {
-        assert :milestone => 1234
+        assert :milestone => 41319
         json :html_url => "the://url"
       }
       """
-    When I successfully run `hub pull-request -m hereyougo -M 1234`
+    When I successfully run `hub pull-request -m hereyougo -M "Hello World!"`
     Then the output should contain exactly "the://url\n"
 
+  Scenario: Pull request fails with unknown milestone before it's created
+    Given I am on the "feature" branch with upstream "origin/feature"
+    Given the GitHub API server:
+      """
+      get('/repos/mislav/coral/milestones') {
+        status 200
+        json [
+          { :number => 1234, :title => "prerelease" },
+          { :number => 1337, :title => "v1" },
+          { :number => 41319, :title => "Hello World!" }
```

Comment:
You can return an empty JSON array here, since we know in advance that no milestones will match "unkown".